source/administration/telemetry.rst

```diff
@@ -77,7 +77,7 @@ Non-personally Identifiable Error Information, distinguished by end users and Sy
 Non-personally Identifiable Diagnostic Information, distinguished by end users and System Admins
   Boolean when the following events occur:
 
-  - *Team and Account Setup Diagnostics:* Account creation via email, invite or UI, account creation page view, account creation completion; tutorial step and tip completion or opt out, team creation page view, team name and URL entry, team creation completion
+  - *Team and Account Setup Diagnostics:* Account creation via email, invite or UI, account creation page view, account creation completion; tutorial step and tip completion or opt out, team creation page view, team name and URL entry, team creation completion. Click events on signup_email, select_team, display_name and team_url pages.
```

Comment:
The events are actually already included
 - Account creation via email
 - team creation page view, team name and URL entry, team creation completion

We can add `select_team` event though. So perhaps:

Reply:
```suggestion
  - *Team and Account Setup Diagnostics:* Account creation via email, invite or UI, account creation page view, account creation completion; tutorial step and tip completion or opt out, team creation page view, team name and URL entry, team creation completion, team selection page.
```

Reply:
Well the change is to track clicks on all form elements, buttons, textboxes and links on these pages, not just the page views.

Reply:
```suggestion
  - *Team and Account Setup Diagnostics:* Account creation via email, invite or UI, account creation page view, account creation completion; tutorial step and tip completion or opt out, team creation page view, team name and URL entry, team creation completion, clicks on all form elements, buttons, textboxes and links on sign up page, team selection page and team creation pages
```

Reply:
Got it, how about this?

Reply:
yep wfm